solr/Dockerfile

```diff
@@ -24,6 +29,15 @@ RUN --mount=type=cache,id=solr-downloads-${TARGETARCH},sharing=locked,target=/op
     && \
     cleanup.sh
 
+RUN --mount=type=cache,id=solr-ocrhighlight-downloads,sharing=locked,target=/opt/downloads \
+    mkdir -p ${OCRHIGHLIGHT_DEST} && \
```

Comment:
Not sure if this is actually needed. I think the download script will do so. If that's the case we can get rid of the variable for the directory since it will only be used on one spot.